store/messagebus_queuemessage.yaml

```diff
@@ -0,0 +1,27 @@
+import:
```

Comment:
Can we split queuemessage (referring to name of the file) so that it is properly converted to CamelCase?